components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher.v1.common/src/main/java/org/wso2/carbon/apimgt/rest/api/publisher/v1/common/mappings/ExportUtils.java

```diff
@@ -875,10 +876,13 @@ public static void addAPIMetaInformationToArchive(String archivePath, APIDTO api
                                 exportFormat, ImportExportConstants.GRAPHQL_COMPLEXITY, graphQLQueryComplexityInfoDTO);
                     }
                 }
+                API api = APIMappingUtil.fromDTOtoAPI(apiDtoToReturn, apiDtoToReturn.getProvider());
+                String organization = ApiMgtDAO.getInstance().getOrganizationByAPIUUID(apiIdentifier.getUUID());
+                api.setOrganization(organization);
+                apiDtoToReturn.setOrganization(organization);
                 // For GraphQL APIs, swagger export is not needed
```

Comment:
addAPIMetaInformationToArchive() is fixed from https://github.com/wso2/carbon-apimgt/pull/10543